analysis/beacon/analyzer.go

```diff
@@ -167,37 +167,37 @@ func (a *analyzer) start() {
 			tsDurationScore := duration
 
 			//smaller data sizes receive a higher score
-			dsSmallnessScore := 1.0 - (float64(dsMode) / 65535.0)
+			dsSmallnessScore := 1.0 - float64(dsMode)/65535.0
 			if dsSmallnessScore < 0 {
 				dsSmallnessScore = 0
 			}
 
-			output := &dataBeacon.BeaconAnalysisOutput{
-				UconnID:           data.uconnID,
-				TS_iSkew:          tsSkew,
-				TS_iDispersion:    tsMadm,
-				TS_duration:       duration,
-				TS_iRange:         tsIntervalRange,
-				TS_iMode:          tsMode,
-				TS_iModeCount:     tsModeCount,
-				TS_intervals:      intervals,
-				TS_intervalCounts: intervalCounts,
-				DS_skew:           dsSkew,
-				DS_dispersion:     dsMadm,
-				DS_range:          dsRange,
-				DS_sizes:          dsSizes,
-				DS_sizeCounts:     dsCounts,
-				DS_mode:           dsMode,
-				DS_modeCount:      dsModeCount,
+			output := &dataBeacon.AnalysisOutput{
+				UconnID:          data.uconnID,
+				TSISkew:          tsSkew,
+				TSIDispersion:    tsMadm,
```

Comment:
Most of these variables had to be modified because go variable names should not contain underscores.